ai-ml/llm-finetuning-gemma/README.md

```diff
@@ -1,3 +1,6 @@
-# Finetune an LLM with multiple GPUs in GKE samples
+# Tutorial: Finetuning Gemma 2b on GKE using L4 GPUs
 
-TBD
+[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine).
+
+Visit [Google Cloud documentation](https://cloud.devsite.corp.google.com/kubernetes-engine/docs/tutorials/finetune-gemma-gpu)
```

Comment:
This should be https://cloud.google.com/kubernetes-engine/docs/tutorials/finetune-gemma-gpu